src/guide/assets.md

```diff
@@ -0,0 +1,289 @@
+# Handling multiple assets
+
+The Liquid sidechain can also be used to send and receive other assets registered on the Liquid Network. Using the SDK you can send and receive these assets by using a Liquid payment with an additional asset ID. By default the SDK includes the metadata for L-BTC and Tether USD. To include addition asset metadata, see [Adding asset metadata](#adding-asset-metadata).
+
+## Adding asset metadata
+
+You can add addition asset metadata to the SDK when you configure it on connect. In the example below we will add the [PEGx EUR](https://assets.blockstream.info/18729918ab4bca843656f08d4dd877bed6641fbd596a0a963abbf199cfeb3cec.json) asset. Once the asset metadata is added, it can be used as an asset to send and receive. 
```

Comment:
I read "It can be use as an asset to send and receive" as if the user doesn't add the metadata than is not able to send/receive. I think we don't block that right? We only use the configuration as metadata don't we?

Reply:
The metadata gives the precision of the asset, so we can't use floating point amounts (e.g. 1.50 USDt) if we don't have the metadata. Without metadata then we can only do send/receive with sat amounts (e.g. 150000000)

Reply:
I see. So just to confirm in that case users might receive but no be able to spend right?

Reply:
Just to correct myself, the issue is only sending. Without the asset metadata a user can prepare/receive an asset payment.